keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/Cluster.kt

```diff
@@ -0,0 +1,49 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import com.netflix.spinnaker.keel.api.ec2.HealthCheckType.EC2
+import com.netflix.spinnaker.keel.api.ec2.TerminationPolicy.OldestInstance
+import java.time.Duration
+
+@JsonInclude(NON_NULL)
+data class Cluster(
+  // what
+  val application: String,
+  val name: String,
+  val imageId: String,
```

Comment:
assumption: at the lowest level we're working with an image rather than trying to determine the blessed version of a package